hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/repository/EntityRepositoryTest.java

```diff
@@ -98,4 +99,30 @@ void entityHistory() {
         assertThat(entityRepository.findAll()).containsExactly(entity);
         assertThat(entityHistory).containsExactly(entity);
     }
+
+    @Test
+    void entityFindByAlias() {
```

Comment:
`entity` prefix for these tests is redundant inside the `EntityRepositoryTest` context.
```suggestion
    void findByAlias() {
```

Reply:
Updated